docs/docs/integrations/llms/yandex.ipynb

```diff
@@ -0,0 +1,135 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# YandexGPT\n",
+    "\n",
+    "This notebook goes over how to use Langchain with [YandexGPT](https://cloud.yandex.com/en/services/yandexgpt).\n",
+    "\n",
+    "To use, you should have the `yandexcloud` python package installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install yandexcloud"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, you should [create service account](https://cloud.yandex.com/en/docs/iam/operations/sa/create) with the `ai.languageModels.user` role.\n",
+    "\n",
+    "Next, you have two authentication options:\n",
+    "- [IAM token](https://cloud.yandex.com/en/docs/iam/operations/iam-token/create-for-sa).\n",
+    "    You can specify the token in a constructor parameter `iam_token` or in an environment variable `YC_IAM_TOKEN`.\n",
+    "- [API key](https://cloud.yandex.com/en/docs/iam/operations/api-key/create)\n",
+    "    You can specify the key in a constructor parameter `api_key` or in an environment variable `YC_API_KEY`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain.chains import LLMChain\n",
+    "from langchain.llms import YandexGPT\n",
+    "from langchain.prompts import PromptTemplate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "template = \"\"\"What is the capital of {country}?\n",
+    "Write, the name of the city mayor in 2001 and the population\n",
+    "\"\"\"\n",
+    "prompt = PromptTemplate(template=template, input_variables=[\"country\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "ValidationError",
```

Comment:
If this error provided for purpose, could you, please, add an error explanation.
Otherwise, you'd rather remove the error(cell output).